File: app/src/main/java/com/cat/novocare/NovoCare.java
package com.cat.novocare;

import androidx.annotation.NonNull;
import com.akexorcist.localizationactivity.ui.LocalizationApplication;
import java.util.Locale;

public class NovoCare extends LocalizationApplication {

    @NonNull
    @Override
    public Locale getDefaultLanguage() {
        return Locale.ENGLISH;
    }
}

File: app/src/main/java/com/cat/novocare/NovoCare.kt
package com.cat.novocare

import android.app.Application
import android.content.Context
import android.content.res.Configuration
import com.akexorcist.localizationactivity.core.LocalizationApplicationDelegate
import java.util.*

class NovoCare: Application() {
    private val localizationDelegate = LocalizationApplicationDelegate()

    override fun attachBaseContext(base: Context) {
        localizationDelegate.setDefaultLanguage(base, Locale.ENGLISH)
        super.attachBaseContext(localizationDelegate.attachBaseContext(base))
    }

    override fun onConfigurationChanged(newConfig: Configuration) {
        super.onConfigurationChanged(newConfig)
        localizationDelegate.onConfigurationChanged(this)
    }

    override fun getApplicationContext(): Context {
        return localizationDelegate.getApplicationContext(super.getApplicationContext())
    }
}